build/gen-tls-certs.sh

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env bash
+
+cat > v3.txt <<- EOF
```

Comment:
I didn't see where this script was invoked from. Are we just using this to generate the cert that goes into the quick start? A comment explaining what this script is used for might be useful.

Reply:
👍 Yup you're right. I find the generation of proper SNI TLS certs hard to remember (to say the least), and thus thought it better to record it like this. I'll add a comment!